test_first.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import requests
+import json
+from app import app
+
+test_browser = app.test_client()
+
+url = 'http://127.0.0.1:5000'
+
```

Comment:
We can delete this `url` line because it is no longer being used.

Reply:
This will work without the server running? great test to start with